src/routes/admin-portal/users/add/+page.svelte

```diff
@@ -7,12 +7,13 @@
 
 	/** Add a new user to the database and the auth system */
 	async function AddUser() {
+		let loweredEmail = email.toLowerCase()
 		const response = await fetch('/api/admin-portal/users', {
 			method: 'POST',
 			headers: {
 				'Content-Type': 'application/json'
 			},
-			body: JSON.stringify({ email, password, name })
+			body: JSON.stringify({ loweredEmail, password, name })
```

Comment:
```suggestion
			body: JSON.stringify({ email: loweredEmail, password, name })
```
My Boiiij you were about to cause another bug with your bugfix